.rubocop_todo.yml

```diff
@@ -11,11 +11,12 @@ Betterment/NonStandardActions:
   Exclude:
     - 'config/routes.rb'
 
-# Offense count: 13
+# Offense count: 14
```

Comment:
Note to self I think this cop can be configured with models that aren't considered "owned" which we should use to knock this exclusion list out.